module/os/freebsd/spl/spl_vfs.c

```diff
@@ -39,6 +39,7 @@ __FBSDID("$FreeBSD$");
 
 #include <sys/mutex.h>
 #include <sys/vnode.h>
+#include <sys/taskq.h>
```

Comment:
Did you mean to include this in the PR?

Reply:
It was implicitly included before. Won't build without.